src/interfaces/profile.interface.ts

```diff
@@ -2,4 +2,12 @@ export interface Profile {
     name: string;
     team: string;
     apiToken: string;
+    authenticationType: string;
```

Comment:
nit: You can use the AuthenticationType here as well, like `authnenticationType: AuthenticationType` :smile:

Reply:
OK I changed it.